src/main/java/net/sf/jabref/sql/DBStrings.java

```diff
@@ -30,14 +34,14 @@
     private String username;
     private String password;
 
-    private String[] serverTypes;
+    private List<String> serverTypes;
     private boolean isInitialized;
     private boolean configValid;
 
 
     /** Creates a new instance of DBStrings */
     public DBStrings() {
-        this.setServerType(null);
+        this.serverTypes = Collections.EMPTY_LIST;
```

Comment:
Collections.emptyList(); should be used instead as it uses the correct generics.